core/reactor_common.c

```diff
@@ -1978,13 +1983,28 @@ void initialize(void) {
     current_tag.time = physical_start_time;
     start_time = current_tag.time;
 
-    LF_PRINT_DEBUG("Start time: %lldns", start_time);
+    #ifdef BIT_32
+        #ifdef MICROSECOND_TIME
+            LF_PRINT_DEBUG("Start time: %ldus", start_time);
+        #else
+            LF_PRINT_DEBUG("Start time: %ldns", start_time);
+        #endif
+    #else
+        #ifdef MICROSECOND_TIME
+            LF_PRINT_DEBUG("Start time: %ldus", start_time);
+        #else
+            LF_PRINT_DEBUG("Start time: %ldns", start_time);
+        #endif
+    #endif
 
+    #ifdef ARDUINO
+    printf("---- Start execution at time %ldus\n", physical_start_time);
```

Comment:
This just prints the microseconds. Shouldn't we factor this into seconds + microseconds, like in the `#else` branch?